lib/chores/populator.rb

```diff
@@ -0,0 +1,177 @@
+require "json"
+
+class Populator
+  def initialize(model, n_records=100)
+    @model = model
+    @n_records = (n_records || 100).to_i
+  end
+
+  def run(n_records=@n_records)
+    method_name = "make_#{@model.to_s.downcase}_record".to_sym
+    unless respond_to?(method_name, true)
```

Comment:
what happens when you don't have this? seems like it would give you a `noMethodError` without it, but guessing the error msg was unclear somehow?